functest/package_test.go

```diff
@@ -106,9 +138,16 @@ func TestPackageBuildahOptions(t *testing.T) {
 	sandbox.ProjectDir = filepath.Join(sandbox.ProjectDir, "starter")
 
 	args := []string{"stack", "package", "--buildah", "--buildah-options", "--format=docker"}
-	_, err = cmdtest.RunAppsody(sandbox, args...)
+	output, err := cmdtest.RunAppsody(sandbox, args...)
 	if err != nil {
 		t.Fatal(err)
+	} else {
+		if !strings.Contains(output, "format=docker") {
```

Comment:
why not include the dashes in the Contains?

Reply:
Don't think the dashes are passed through in the Buildah command